src/app/dock_widgets/process/gt_processdock.cpp

```diff
@@ -1474,7 +1475,13 @@ GtProcessDock::moveElements(const QList<QModelIndex>& source,
 
     if (!targetComp) return;
 
-    auto moveCmd = gtApp->makeCommand(m_taskGroup,
+
+    auto* commonParent = gt::find_lowest_ancestor(objectsToMove,
```

Comment:
I think you may need target component here as well (You should be able to use the << operator to create temporary list). This is the object where the tasks are moved to, correct?
```suggestion
    auto* commonParent = gt::find_lowest_ancestor(objectsToMove << targetComp,
```

Reply:
May check if this is indeed needed?